### PR TITLE
feat: add check-node-version script

### PR DESCRIPTION
**Pull request by dtjones404**
* Added `check-node-version` script
* Added `.nvmrc`, which uses Node version `16.8`
* Added a few lines to Readme about Node version requirements

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v16.8
```
Comment (gregjopa) [inline .nvmrc:1]: Nit: please add a new line at the end of the file

File: README.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ PAYPAL_API_BASE_URL=https://api-m.sandbox.paypal.com
 PAYPAL_WEB_BASE_URL=https://www.paypal.com
 ```
 
+This application requires Node.js version 16.8.0 or higher. Ensure that the correct Node.js version is being used by running:
```
Comment (gregjopa) [inline README.md:16]:
```suggestion
This application requires Node.js version 16.8.0 or higher. 
```

Could we remove the `nvm` command? Technically there are other version managers so I don't want to assume everyone is using nvm. For example, I like using [volta](https://docs.volta.sh/guide/)

**Review by gregjopa — APPROVED** [review]
💯

File: .nvmrc (hunk 1 above)
Comment (dtjones404) [reply .nvmrc:1]: Ah good point!  Done.

File: README.md (hunk 2 above)
Comment (dtjones404) [reply README.md]: Ah good point!  Done.

**Pull request merged**